### PR TITLE
Support PostgreSQL 17

### PR DESCRIPTION
**Pull request by hlinnaka**
These seem to be the only changes required.

### CODE REVIEW

**Comment by JelteF** [comment]
Locally this works for me. But I had to modify the regression Makefile a tiny bit to allow me to run the tests locally. Let's wait for #124 to be merged though before merging this. To be sure that CI still succeeds after these changes.

**Review by JelteF — APPROVED** [review]

**Review by wuputah — APPROVED** [review]

**Pull request merged**
